setup.py

```diff
@@ -20,7 +20,7 @@
 
 install_requires = [
     "tqdm >= 4.14",
-    "pkginfo >= 1.0",
+    "pkginfo >= 1.4.2",
     "requests >= 2.5.0, != 2.15, != 2.16",
     "requests-toolbelt >= 0.8.0",
     "setuptools >= 0.7.0",
```

Comment:
Or rather, even if we don't need to since the new additional metadata is optional, should we take this opportunity to do some kind of required version increase regardless?

Reply:
The only reason we have setuptools listed in the requirements is so that we can reliably get it's version when running `twine --version`.

Users with older versions of `setuptools` will still be able to upload their distributions with twine, they just won't be able to create distributions with Metadata 2.1 fields. If they want to do that, they should upgrade their `setuptools` (it's not twine's responsibility to make that decision for them).

Reply:
Totally understand. Thanks. So, if we choose to do any user-facing messaging about this feature &, for instance, Markdown support, we should indicate that such-and-such `setuptools` is a requirement.